editoast/src/schema/errors.rs

```diff
@@ -4,14 +4,13 @@ use serde::{Deserialize, Serialize};
 use strum_macros::EnumVariantNames;
 
 #[derive(Serialize, Deserialize, PartialEq, Debug, Clone)]
-#[serde(deny_unknown_fields)]
 pub struct InfraError {
     obj_id: String,
     obj_type: ObjectType,
     field: Option<String>,
     is_warning: bool,
     #[serde(flatten)]
-    sub_type: InfraErrorType,
+    error_type: InfraErrorType,
```

Comment:
Is there a reason for this renaming ?

Reply:
This field is stored in the database under the name `error_type`.